src/pc/controller/controller_entry_point.c

```diff
@@ -41,13 +46,40 @@ s32 osContStartReadData(UNUSED OSMesgQueue *mesg) {
     return 0;
 }
 
+
+#define DEBUG_PAD_INPUTS 1
```

Comment:
We we want this define on by default?

Reply:
No, but I don't think I have the calibration correct yet, hence why it is still 1.